src/redemptions/redemption.service.ts

```diff
@@ -430,4 +462,7 @@ export class RedemptionService {
 
     return { attemptable: true, reason: '' };
   }
+
+  minorAgeMessage = (age: number): string =>
```

Comment:
@danield9tqh this message will get piped into `can_attempt_reason` and `can_attempt=true`